.gitignore

```diff
@@ -24,5 +24,6 @@ out/
 
 ### App ###
 src/main/frontend/*
+!/frontend/themes/
```

Comment:
Oops, good catch! I think I cut this out when I was thinking I didn't have to write any css, but forgot to put it back.